autocomplete/definitions/namedTypes/tes3baseObject/__tojson.lua

```diff
@@ -0,0 +1,6 @@
+return {
+	type = "method",
+	description = [[Serializes the object to json.]],
+	returns = "string",
```

Comment:
The preferred way to declare return values now is:
```lua
returns = {
    { name = "userVisibleNameNoSpaces", type = "type1|type2|type3", description = "User-facing description." },
},
```
This is more flexible than the weird mixture of `valuetype` and `returns` being combined. Backwards support is provided for that, and it's fine here, just know that I'm planning on moving away from that format in the future.

Reply:
Cool. It should give more information to the reader.